src/main/java/com/coveros/selenified/element/check/azzert/AssertContains.java

```diff
@@ -0,0 +1,243 @@
+ * 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.coveros.selenified.element.check.azzert;
+
+import com.coveros.selenified.OutputFile;
+import com.coveros.selenified.element.Element;
+import com.coveros.selenified.element.check.Contains;
+
+import java.util.Arrays;
+
+import static com.coveros.selenified.element.check.Constants.ELEMENT_NOT_SELECT;
+import static com.coveros.selenified.element.check.Constants.NO_ELEMENT_FOUND;
+import static org.testng.AssertJUnit.*;
+
+/**
+ * Contains extends Asserts to provide some additional verification
+ * capabilities. It will handle all verifications performed on the actual
+ * element. These asserts are custom to the framework, and in addition to
+ * providing easy object oriented capabilities, they take screenshots with each
+ * verification to provide additional traceability, and assist in
+ * troubleshooting and debugging failing tests. Contains checks that elements
+ * have a particular value associated to them.
+ *
+ * @author Max Saperstone
+ * @version 3.1.0
+ * @lastupdate 2/22/2018
+ */
+public class AssertContains implements Contains {
+
+    // this will be the name of the file we write all commands out to
+    private final OutputFile file;
+
+    // this is the element that all actions will be performed on
+    private final Element element;
+
+    public AssertContains(Element element, OutputFile file) {
+        this.element = element;
+        this.file = file;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public OutputFile getOutputFile() {
+        return file;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Element getElement() {
+        return element;
+    }
+
+    // ///////////////////////////////////////
+    // assessing functionality
+    // ///////////////////////////////////////
+
+    /**
+     * Verifies that the element's class contains the provided expected class.
+     * If the element isn't present, this will constitute a failure, same as a
+     * mismatch. This information will be logged and recorded, with a screenshot
+     * for traceability and added debugging support.
+     *
+     * @param expectedClass - the expected class value
+     */
+    public void clazz(String expectedClass) {
+        String clazz = checkClazz(expectedClass, 0, 0);
+        assertNotNull(NO_ELEMENT_FOUND, clazz);
+        assertTrue("Class Mismatch: class of '" + clazz + "' doesn't contain '" + expectedClass + "'", clazz.contains(expectedClass));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "' doesn't contain '" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)